lib/misc/cells.js

```diff
@@ -0,0 +1,64 @@
+(function() {
+  var scopes, values;
+
+  scopes = require('./scopes');
+
+  values = require('underscore-plus').values;
+
+  module.exports = {
+    getRange: function(ed) {
+      var buffer, cursor, end, regex, regexString, start;
+      buffer = ed.getBuffer();
+      start = buffer.getFirstPosition();
+      end = buffer.getEndPosition();
+      regexString = atom.config.get('julia-client.cellDelimiter');
+      regex = new RegExp(regexString);
+      cursor = ed.getCursorBufferPosition();
+      if (cursor.row > 0) {
+        buffer.backwardsScanInRange(regex, [start, cursor], function(arg) {
+          var range;
+          range = arg.range;
+          return start = range.start;
+        });
+      }
+      buffer.scanInRange(regex, [cursor, end], function(arg) {
+        var range;
+        range = arg.range;
+        return end = range.start;
+      });
+      return [values(start), values(end)];
+    },
+    get: function(ed) {
+      var range, res;
+      range = this.getRange(ed);
+      res = {
+        range: range,
+        selection: ed.getSelections()[0],
+        line: range[1][1] + 1,
```

Comment:
`range[1][0]` is the line, while `[1][1]` is the column. the `+1` is also unnecessary, since that is handled in `eval`.